pkg/cmd/server/origin/master.go

```diff
@@ -390,10 +393,24 @@ func (c *MasterConfig) InitializeObjects() {
 	c.ensureOpenShiftSharedResourcesNamespace()
 }
 
-func (c *MasterConfig) InstallProtectedAPI(apiContainer *genericmux.APIContainer) ([]string, error) {
+func (c *MasterConfig) InstallProtectedAPI(apiserver *genericapiserver.GenericAPIServer) ([]string, error) {
+	apiContainer := apiserver.HandlerContainer
+
 	// initialize OpenShift API
 	storage := c.GetRestStorage()
 
+	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(buildapi.GroupName)
+	//if apiResourceConfigSource.AnyResourcesForVersionEnabled(rbacapiv1alpha1.SchemeGroupVersion) {
+	apiGroupInfo.VersionedResourcesStorageMap[buildapiv1.SchemeGroupVersion.Version] = map[string]rest.Storage{
```

Comment:
Not that this has to do it, but I like the pattern I made upstream with these being built closer to the registry.  I don't want  the interface I made for it (that was a mistake), but the code locality worked out nicely.